src/hotspot/share/code/codeBlob.hpp

```diff
@@ -290,6 +303,9 @@ class RuntimeBlob : public CodeBlob {
 
   // Deal with Disassembler, VTune, Forte, JvmtiExport, MemoryService.
   static void trace_new_stub(RuntimeBlob* blob, const char* name1, const char* name2 = "");
+
+  class Vptr : public CodeBlob::Vptr {
+  };
```

Comment:
Was this needed for some compiler? Or is it to be more explicit about the type hierarchy?

Reply:
Thank you, @xmas92, for review and suggestions.

It is second (explicit type hierarchy). I think it should be explicitly declared (even empty) because it is referenced in subclasses to avoid confusion. And it could be useful in a future if we need other virtual methods.

Local build with `gcc` on Linux passed without it but  I did not try to build on other platforms.